### DISCUSSION TITLE
Hosting database on a different instance

### DISCUSSION THREAD

**Discussion by HecFranco** [symfony/symfony#54722] [Q&A]
We are migrating a dockerised symfony project to an infra cloud, and we have gone from having all the core project and database on the same host, to having the database on a different specialized instance but on another machine. When doing this we detected that the response time of TraceableFirewallListener (a security component) increases. Do you know how to configure symfony and doctrine so that the connection to the database is not on the same machine?

**Comment by Seb33300** [comment]
Do you have any query triggered by that Listener?
If you move your database from local to an external server, you will have network latency and I dont think there is anything you can do about this except optimizing your application.
I had a similar issue recently after moving the local database of a Laravel application to a dedicated database server.
Page load increased to more than 3s and after investigation I found out that we had more than 1000 queries per page, each of them taking 0.005s due to network latency, when it was insignifiant before with a local database. 
This was a misconception in the security component to handle users permissions. We made a huge improvement and reduced the number of queries to ~20... And this solved our issue.
